test/controllers/auth_controller_test.exs

```diff
@@ -5,9 +5,8 @@ defmodule ConstableWeb.AuthControllerTest do
   require Constable.Pact
 
   @google_authorize_url "https://accounts.google.com/o/oauth2/auth"
-  @permitted_email_domain Application.fetch_env!(:constable, :permitted_email_domain)
-
-  def valid_email_address, do: "fake@#{@permitted_email_domain}"
+  def valid_email_address, do: "fake@#{permitted_email_domain()}"
```

Comment:
I didn't use a private function here, cause it looks like it's used like it must be public - in `AuthControllerTest.valid_email_address()`, a few lines down